src/neural/opencl/network_opencl.cc

```diff
@@ -245,8 +246,11 @@ class OpenCLNetwork : public Network {
     params_.tune_only = options.GetOrDefault<bool>("tune_only", false);
     params_.tune_exhaustive =
         options.GetOrDefault<bool>("tune_exhaustive", false);
-    params_.tuner_file =
-        options.GetOrDefault<std::string>("tuner_file", "leelaz_opencl_tuning");
+    if (options.IsDefault<std::string>("tuner_file")) {
+      params_.tuner_file = GetUserCacheDirectory() + "lc0/leelaz_opencl_tuning";
```

Comment:
This means multiple installs of lc0 get the same tuner file unless you use backend-opts to override?
I wonder if that will ever cause us pain...

Reply:
hmm only on linux it seems - windows this is empty string, so maybe I am fine with that :P